src/hooks/useFetch.js

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from 'react';
+
+const fetchUrl = (url, after, count, limit = 100, time = 'year') => {
+  const urlParams = `t=${time}&limit=${limit}${
```

Comment:
Having a ternary inside this template string makes it hard to read. I'd suggest to use an if clause here.

```
let urlParams = `t=${time}&limit=${limit}`;
if (after) {
  urlParams += `&after=${after}`
}
```